csv_datasets/random_graph/csv/e-0.csv

```diff
@@ -0,0 +1 @@
+5,0,0,-2057845723,0.6007943693210459,0.8075239484603391,True,88,26059,K,UIYRTUxugm,2192-02-20,2116-08-08T06:15:52.087+00:00
```

Comment:
When it comes to naming it is not an easy task.

I suggest we have a directory `random_graphs` and then keep this graph(s) you have as a subdirectory, because we are likely to have a lot more included over time.

Reply:
You can keep the original name from the generator.

Reply:
Done